lib/ghee/api/issues.rb

```diff
@@ -126,6 +126,42 @@ def events(id=nil)
             Ghee::API::Repos::Issues::Events::Proxy.new(connection,prefix)
           end
 
+          def assignees(&block)
+            prefix = "#{path_prefix}/assignees"
+            Ghee::API::Repos::Issues::Assignees::Proxy.new(connection, prefix, &block)
+          end
+        end
+
+        module Assignees
+          class Proxy < ::Ghee::ResourceProxy
+            accept_header "application/vnd.github.cerberus-preview.full+json"
```

Comment:
In the past we have had Ghee abstract it. That way an upgrade of Ghee will just fix itself 

I'm pretty sure that GitHub still honors the preview vnc after shipping the feature into the API, so I think it's pretty safe.